### PR TITLE
XW-4299 | replace wds-icons-play with updated svg

### PR DESCRIPTION
**Pull request by rybmat**
@Wikia/x-wing

### CODE REVIEW

**Comment by vforge** [comment]
@rybmat Previously icons didn't have color and were monochromatic and this one has two defined colors. It slightly changes how the icon can be used and styled. Could we maybe not replace the current one and put this one as separate *asset* (instead of icon)?

File: style-guide/styles/wds-components/_player.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.wds-player-icon-play-tiny {
+	$size: 1.2 * 30px;
```
Comment (latata) [inline style-guide/styles/wds-components/_player.scss]: do we have to multiply by `1.2`? can't we just have `36px`?

File: style-guide/styles/wds-components/_player.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+	height: $size;
```
Comment (latata) [inline style-guide/styles/wds-components/_player.scss]: how about using `square` mixin?

File: dist/scss/wds-components/_player.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
+.wds-player-icon-play-tiny {
+	$size: 1.2 * 30px;
+
+	height: $size;
```
Comment (hakubo) [inline dist/scss/wds-components/_player.scss]: you can use square mixin

Comment (hakubo) [reply dist/scss/wds-components/_player.scss]: also SCSS support loop over map so you could generate these clasess
http://thesassway.com/intermediate/if-for-each-while

File: style-guide/styles/wds-components/_player.scss (hunk 2 above)
Comment (rybmat) [reply style-guide/styles/wds-components/_player.scss]: 👍

File: style-guide/styles/wds-components/_player.scss (hunk 1 above)
Comment (rybmat) [reply style-guide/styles/wds-components/_player.scss]: @latata I thought it would be easier to understand the logic behind it

**Pull request merged**